### ISSUE TITLE
Multilabel Image classification 

### ISSUE THREAD

**Issue by abhigoku10** [lukemelas/PyTorch-Pretrained-ViT#21]
@lukemelas thanks for sharing code base , i had few queries
1.can we train crossvit for multilabel classification problem , if so what is the procedure
2. i have a custom dataset of 10.5k with 25class labels with instance as label vectors of 0 and 1
3. can remove the pre-trained classifer head and add our customr classifier ?

Thanks in advance
